tests/osgi-tests/pom.xml

```diff
@@ -167,6 +167,24 @@
             <type>zip</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.ops4j.pax.exam</groupId>
+            <artifactId>pax-exam-junit4</artifactId>
+            <version>3.6.0</version>
```

Comment:
Please move all dependencies to parent poms add the versions in properties section and then refer them here.